telemetry/collector/src/main/java/ru/yandex/practicum/model/hub/ScenarioAddedEvent.java

```diff
@@ -0,0 +1,34 @@
+package ru.yandex.practicum.model.hub;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString(callSuper = true)
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class ScenarioAddedEvent extends HubEvent {
+
+    @NotBlank
+    @Size(min = 3, max = 2147483647)
```

Comment:
max - лишний, так как это его дефолтное значение
```suggestion
    @Size(min = 3)
```

Reply:
Поправил